integrations/helm/release/utils_test.go

```diff
@@ -0,0 +1,168 @@
+package release
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	ifv1 "github.com/weaveworks/flux/apis/helm.integrations.flux.weave.works/v1alpha"
+)
+
+var noopLogger = log.NewNopLogger()
+
+type mappifyTestCase struct {
+	desc     string
+	param    string
+	value    string
+	expected map[string]interface{}
+}
+
+type collectTestCase struct {
+	desc     string
+	params   []ifv1.HelmChartParam
+	expected map[string]interface{}
+}
+
+type unwrapTestCase struct {
+	desc     string
+	value    string
+	expected []interface{}
+}
+
+// fixtures for mappoifyValueOverride tests
+var mappifyExpectedSimple = map[string]interface{}{
```

Comment:
these fixtures can all be `const`s

Reply:
Constants can only be numbers, characters (runes), strings or booleans.

Reply:
yep, sorry this was wrong

Reply:
no worries :+1: